template/rwa-onboarding/Goerli-DssSpell.t.sol

```diff
@@ -20,466 +18,834 @@ pragma solidity 0.6.12;
 
 import "./Goerli-DssSpell.t.base.sol";
 
+interface RwaLiquidationLike {
+    function ilks(bytes32) external returns (string memory, address, uint48 toc, uint48 tau);
+    function bump(bytes32 ilk, uint256 val) external;
+    function tell(bytes32) external;
+    function cure(bytes32) external;
+    function cull(bytes32, address) external;
+    function good(bytes32) external view returns (bool);
+}
+
+interface RwaUrnLike {
+    function can(address) external view returns (uint256);
+    function gemJoin() external view returns (GemAbstract);
+    function lock(uint256) external;
+    function draw(uint256) external;
+    function wipe(uint256) external;
+    function free(uint256) external;
+}
+
+interface RwaOutputConduitLike {
+    function can(address) external view returns (uint256);
+    function may(address) external view returns (uint256);
+    function gem() external view returns (GemAbstract);
+    function bud(address) external view returns (uint256);
+    function pick(address) external;
+    function push() external;
+    function push(uint256) external;
+    function quit() external;
+    function kiss(address) external;
+    function mate(address) external;
+    function hope(address) external;
+    function quitTo() external view returns (address);
+}
+
+interface RwaInputConduitLike {
+    function wards(address) external view returns (uint256);
+    function may(address) external view returns (uint256);
+    function quitTo() external view returns (address);
+    function mate(address) external;
+    function push() external;
+}
```

Comment:
[This PR](https://github.com/makerdao/dss-interfaces/pull/75) has just been merged, so it would be good to update `dss-interfaces` dependency and reference the contracts from [this directory](https://github.com/makerdao/dss-interfaces/tree/master/src/dss/mip21) instead of re-declaring them.